chord_metadata_service/chord/tests/test_api_search.py

```diff
@@ -49,7 +49,7 @@ def test_data_type_list(self):
         r = self.client.get(reverse("data-type-list"))
         self.assertEqual(r.status_code, status.HTTP_200_OK)
         c = r.json()
-        self.assertEqual(len(c), 3)
+        self.assertEqual(len(c), 4)
```

Comment:
if there isnt any object on the other end actually stored in katsu, the data type should be excluded from search

Reply:
Hmm I think this might cause bugs in the front end though - since it's not a "real" data type (you can't search for objects that fit the schema or anything right?)

Reply:
For the purposes of the workflow it's a data type, but since it's just a workaround to ingest files into DRS I think it shouldn't be listed by the service except in the workflow part- at least that's my intuition

Reply:
the search drop-down data types are fetched from this list of data types, right?

Reply:
I think so

Reply:
I am wondering if it's the same data types list that also serves the drop-down in Data manager when creating the table - there we would need it to specify the data type for ingestion.